gmgc_mapper_plotter/GMGC_Mapper_plotter.ipynb

```diff
@@ -0,0 +1,222 @@
+NoanbL6k7L6k7L6k7I2quTqMUzoMbaunby+Rw0ziyE4SHvHOfd+re3RwEjg/5ztWOfcTGAmnL3W58WoqKxkZ1ER5ZWVHC0rY1tBAZkpKeQeOQLAkE6dAHh30yZW7NtHelISGa1acaysjJLyctonJrJwxw4emj+fJ6+/nlvS0yk6cYJWl10GwN6jR9l++DCvdO5MWSDAhvx8TlRUEKiqIjd0CfW65OQ6+3b0aPB2vZYtWzZU3IilrP6krP6krP6krI2qoD6VCbye9WnAG8BW59yU096+DdjmnNsf7n7lFReTOW0aXxYWMn/bNjKnTWP4tdeyfO9eyisruaZNG7KGDeOZxYtZkpvLW/fcQzMzpuTk0GXqVADe3byZiqoqslaupENWFuMWLqxpf+LSpTx/880APDdoEPfOm8cjffqQFB/PktxcOrdqRY+2bevsW0lJCSUlJY3/lxABlNWflNWflNWflNV7ntb6NLNBwKfAJqB6xdoJzrk/mtlsYKVzbkZ92urVq5drzKLsZYEAnV95hal33cWIzMxG+5xeM2Ywtm9fHh8woM738/LyAEhLS6vzfT9RVn9SVn9SVn9S1sbTJGp9OueWc5bbsZxzD4W3N+cWFx3N/qefbvTP2fDYY43+GSIiItI0XBp3CIqIiIg0QRqoiYiIiEQoT+9Ra0hmVgxs97of0uCSgboXlZOmTN+rP+l79Sd9r43jKudcyvl28nx5jga0vT435UnTYmZr9L36j75Xf9L36k/6Xr2lS58iIiIiEUoDNREREZEI5aeB2kyvOyCNQt+rP+l79Sd9r/6k79VDvplMICIiIuI3fjqjJiIiIuIrTX6gZmZ3mtl2M9thZs953R+5OGbWwcz+bGZbzWyLmT0Z2t7azBab2ZehP1t53Ve5cGYWZWZ/NbOPQq/TzWxV6Huda2axXvdRLpyZJZnZ78xsW+i3O1C/2abPzMaF/h3ebGbvmlm8frPeadIDNTOLAqYBdwFdgVFm1tXbXslFCgDjnXOZwA3A46Hv8jlgiXPuGmBJ6LU0PU8CW2u9/g8gK/S9FgGPeNIr+aZeBhY4564DehH8jvWbbcLMLA34Z6Cfc647EAV8H/1mPdOkB2rAAGCHc26Xc64c+G9guMd9kovgnPvaObcu9LyY4D/4aQS/z7dCu70F3ONND+VimdmVwHeAWaHXBtwK/C60i77XJsjMLgduAt4AcM6VO+eOoN+sH0QDl5lZNNAc+Br9Zj3T1AdqacC+Wq/3h7ZJE2ZmnYA+wCqgrXPuawgO5oArvOuZXKRfAz8GqkKv2wBHnHOB0Gv9bpumDOAQ8F+hy9qzzKwF+s02ac65PGAysJfgAO0osBb9Zj3T1AdqVsc2TWNtwswsAXgPeMo5d8zr/sg3Y2Z3Awedc2trb65jV/1um55ooC8w3TnXByhFlzmbvNA9hcOBdKA90ILg7UWn0282TJr6QG0/0KHW6yuBrzzqi3xDZhZDcJD2jnPu/dDmfDNrF3q/HXDQq/7JRfkW8F0z203w1oRbCZ5hSwpdVgH9bpuq/cB+59yq0OvfERy46TfbtN0G5DrnDjnnKoD3gRvRb9YzTX2g9r/ANaHZKLEEb3j8wOM+yUUI3bf0BrDVOTel1lsfAKNDz0cD2eHum1w859xPnXNXOuc6Efx9fuKc+wHwZ+C+0G76Xpsg59wBYJ+ZXRvaNBT4HP1mm7q9wA1m1jz073L196rfrEea/IK3ZvZtgv9DjwLedM5N8rhLchHMbBDwKbCJv93LNIHgfWrzgI4E/wH5nnOu0JNOyjdiZkOAZ5xzd5tZBsEzbK2BvwI/dM6Vedk/uXBm1pvgJJFYYBcwhuAJAP1mmzAzmwjcT3A2/l+BfyB4T5p+sx5o8gM1EREREb9q6pc+RURERHxLAzURERGRCKWBmoiIiEiE0kBNREREJEJpoCYiIiISoTRQExEREYlQGqiJiIiIRCgN1EREREQilAZqIiIiIhFKAzURERGRCKWBmoiIiEiE0kBNREREJEJpoCYivmFm3zKzD83ssJmdNLOdZjbVzGLD9PmzzcyZ2S/C8Xki4n8aqImIL5jZ94G/AHcD+4DfALuAx4DmHnZNROSiaaAmIk2emTUHpgFRwBygr3PuUefc7cB1wHEz62lmC8yswMwOhc68XVurDRd6dAq9/kXo9ezQ64dCr5ebWZaZHTGzPDP7Qej92cDoUHPPVx9rZrFm9rqZHTCzMjPbZ2YfhOdvRkSaumivOyAi0gC+BbQOPf+lc66q+g3n3E4za0fwbFsS8AcgluCZt35m1tU5V3SBnxULrAZuB14zsw+BRcAAIBNYBawM7fN/gX8AtgDvA+2BwReZU0QuMTqjJiJ+cEWt53vqeP9BgoO0pc65u51zdwDrgVTgexf4WYXATcB3gEqgBdDFOfdbggMzgAXOuadC22JC2zYB7wBjTuuviMhZaaAmIn5wsNbzq+p4v1Poz621tm07x/4QvIxal63OuZPOuQqgNLQt4Rx9exuYBwwHlgOHgY/NrMU5jhERATRQExF/WAFUX778FzOr+bfNzK7ib2fZrqt1TPX9adXvHQ/9eXnoz+5n+axArefutPcqQ3/W/rc14Jy7P9RuJvAngpdMR56lfRGRGrpHTUSaPOdcqZk9QfDs1Q+BHma2muD9YLcD6cBPgVtCN/LHAn2AfOB3oWb+SvD+s1fNbDvBM2AXal/ozx+aWUtgPnCVmf0EWAOUAD1C+xy5iPZF5BKjM2oi4gvOuXeAW4A/Ah0JzsDMBF4HCkLvLSI4GOtHcFLBLc65wlATTxC8j6w3cCXwXxfRjdcJnt1LA/4Z+D/A9tDnfxt4BCgHfgl8dBHti8glxpw7/cy9iIiIiEQCnVETERERiVAaqImIiIhEKA3URERERCKUBmoiIiIiEUoDNREREZEIpYGaiIiISITSQE1EREQkQmmgJiIiIhKhNFATERERiVD/D7E30q1UK0bdAAAAAElFTkSuQmCC\n",
```

Comment:
You should not commit the output because will likely not match what the user has. Do `Clear all output` & then save again.